compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BXMLType.java

```diff
@@ -26,8 +27,26 @@
  */
 public class BXMLType extends BBuiltInRefType {
 
+    public BType constraint;
+    public boolean defaultConstraint;
+
     public BXMLType(int tag, BTypeSymbol tsymbol) {
         super(tag, tsymbol);
+        this.defaultConstraint = true;
+    }
+
+    public BXMLType(int tag, BType constraint, boolean defaultConstraint, BTypeSymbol tsymbol) {
```

Comment:
First arg is alway XML tag, we don't need this, shall we remove it.

Reply:
Removed.